service/Cargo.toml

```diff
@@ -10,39 +10,18 @@ homepage = "https://www.shuttle.rs"
 doctest = false
 
 [dependencies]
-actix-web = { version = "4.2.1", optional = true }
 anyhow = { workspace = true }
 async-trait = { workspace = true }
 axum = { workspace = true, optional = true }
-bincode = { version = "1.3.3", optional = true }
 # TODO: debug the libgit2-sys conflict with cargo-edit when upgrading cargo to 0.66
 cargo = { version = "0.65.0", optional = true }
 cargo_metadata = "0.15.2"
 crossbeam-channel = "0.5.6"
-hyper = { version = "0.14.23", features = ["server", "tcp", "http1"], optional = true }
-num_cpus = { version = "1.14.0", optional = true }
 pipe = "0.4.0"
-poem = { version = "1.3.49", optional = true }
-rocket = { version = "0.5.0-rc.2", optional = true }
-salvo = { version = "0.37.5", optional = true }
 serde_json = { workspace = true }
-serenity = { version = "0.11.5", default-features = false, features = ["client", "gateway", "rustls_backend", "model"], optional = true }
-poise = { version = "0.5.2", optional = true }
 thiserror = { workspace = true }
-thruster = { version = "1.3.0", optional = true }
-tide = { version = "0.16.0", optional = true }
-tokio = { version = "1.22.0", features = ["sync"] }
-tower = { version = "0.4.13", features = ["make"], optional = true }
+tokio = { version = "1.26.0", features = ["sync"] }
 tracing = { workspace = true }
-tracing-subscriber = { workspace = true, features = ["env-filter"] }
-warp = { version = "0.3.3", optional = true }
-
-# Tide does not have tokio support. So make sure async-std is compatible with tokio
-# https://github.com/http-rs/tide/issues/791
-[dependencies.async-std]
-version = "1.12.0"
-optional = true
-features = ["tokio1"]
```

Comment:
This is just beautiful :cry:

Reply:
Yes, so many good deletes in this release :heart: